tools/localazy/importSupportedLocalesFromLocalazy.py

```diff
@@ -27,6 +27,8 @@ def normalizeForResourceConfigurations(locale):
             return "in"
         case "zh_TW#Hant":
             return "zh-rTW"
+        case "zh#Hans":
+            return "zh-rCN"
```

Comment:
Localazy will import string for simplified Chinese in the folders `values-zh`, and not `values-zh-rCN. I am not sure this need to match the value here.